code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+---
+openapi: 3.0.3
+############################################################################
+#                                     API info                             #
+############################################################################
+info:
+  title: Traffic Influence API
+  version: wip
+  description: |
+    ## Overview
+    The reference scenario foresees a Service, composed by one or more Service
+    Producers deployed in different geographical locations on Edge Cloud Zones
+    (Edge datacentres of Telco Operator) or in Cloud. The Service Producer,
+    deployed at the Edge, is referred as Edge Application Server (EAS).\
+    An Edge Cloud Zone is a platform in the Telco Operator network, offering
+    network, compute and storage resources to developers. A developer can
+    deploy and run applications on an Edge Cloud Zone, meaning reduced latency
+    to end users that are nearby, as the network path is shorter. A network
+    operator's EdgeCloud may comprise multiple Edge Cloud Zones, each in a
+    discrete location to bring latency benefits to end users across a country.\
+    The operator can help developers know which of the Edge Cloud Zones will
+    bring the best performance benefit for a given end user and application\.
+    The Traffic Influence API (TI API) provides the fastest routing from the
+    user Device (e.g. a Smartphone) to the optimal EAS instance in a specific
+    geographical location, installed in an Edge Cloud Zone.\
```

Comment:
I think the API tries to provide the fastest routing via an EAS that may be optimum to serve user Device in specific geographical location but it itself may not be located in the same geographical location as the text is suggesting. If the observation is correct we may need to rephrase the statement a bit.